packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5225,18 +5212,25 @@ export default class Meeting extends StatelessWebexPlugin {
     this.mediaProperties.setMediaPeerConnection(mc);
     this.setupMediaConnectionListeners();
 
-    // publish the tracks
-    if (this.mediaProperties.audioTrack) {
-      await this.publishTrack(this.mediaProperties.audioTrack);
+    if (this.isMultistream) {
+      this.sendSlotManager.createSlot(mc, MediaType.VideoMain, false);
```

Comment:
don't we need to set the `active` parameter value based on the values of `audioEnabled`/`videoEnabled` parameters that client passed to `addMedia()`? also not sure about the hardcoded `false` for slides...

Reply:
@marcin-bazyl - Do you remember? We decided that we will createSlot for all four Media types here but it will be disabled by default and later updateMedia shall enable it as per the need. 

That is why we do a `createSlot` here with an inactive state for each sendSlot.

Reply:
yeah I remember we decided that we'll always create the send slots, but the initial "active" value for them should be set correctly based on what the app passes to `addMedia()`. There is no reason to require the app to call `updateMedia()` later. The app will only call `updateMedia()` when it wants to change the values for audioEnabled/videoEnabled to somehing different than what was in the `addMedia()` call

Reply:
Ok. Makes sense. Will update.